### ISSUE TITLE
Spyder Report bug does not link to github

### ISSUE THREAD

**Issue by phsdv** [spyder-ide/spyder#2191]
Spyder Version:  2.3.2
Python Version:  3.4.0
Qt Version    :  4.8.6, PyQt4 (API v2) 4.11.3 on Windows
pyflakes >=0.6.0:  0.8.1 (OK)
pep8 >=0.6      :  1.6.2 (OK)
IPython >=0.13  :  2.4.1 (OK)
zmq >=2.1.11    :  None (NOK)
pygments >=1.6  :  2.0.2 (OK)
pandas >=0.13.1 :  0.15.2 (OK)
sphinx >=0.6.6  :  1.2.3 (OK)
rope >=0.9.2    :  None (NOK)
matplotlib >=1.0:  1.4.2 (OK)
pylint >=0.25   :  1.4.1 (OK)

What steps will reproduce the problem?
1. Goto Help -> Report Issue..

What is the expected output? What do you see instead?
Expected to get directed to GitHub as this is apparently the system to use.
A browser window with following URL is opened instead: https://code.google.com/p/spyderlib/issues/entry?comment=Spyder%20.... [truncated]

**Comment by goanpeca** [comment]
Hi @phsdv, thanks for reporting this, but it has been fixed already on #2182 and will be updated on the next release of spyder.

If you want to try this for yourself you can download the development version and run it like

```
$ git clone https://github.com/spyder-ide/spyder.git
$ cd spyder
$ python bootstrap.py
```

Cheers

**Comment by ccordoba12** [comment]
Yes, we can change things on past releases :-)
